src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -118,6 +124,28 @@ def _list_devices(self, cmd):
 
         return 0, result, ""
 
+    def _list_services(self, cmd):
+        svc_type = cmd.get('svc_type', None)
+
```

Comment:
Line 129 is actually the only difference between `_list_services` and `_service_status` (when ignoring variable renames, unifying `list_services` with `describe_service` and ignoring comments)

![image](https://user-images.githubusercontent.com/2574405/48263774-8752ad00-e427-11e8-9b8a-cbb5e55a564b.png)

Is it possible to use `_service_status` here?